packages/template-retail-react-app/app/pages/product-detail/index.jsx

```diff
@@ -412,8 +412,7 @@ const ProductDetail = () => {
             if (
                 hasPickupItems &&
                 basketResponse?.basketId &&
-                basketResponse.shipments.length > 0 &&
-                !basketResponse.shipments[0].shippingMethod
```

Comment:
If I remove this, do I need to add a check to see if the basket was empty before adding to cart?